src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,18 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
 
+        Machine excavator = new Excavator();
+
+        Machine bulldozer = new Bulldozer();
+
+        Machine truck = new Truck();
+
+        Machine[] machines = new Machine[] {excavator,bulldozer,truck};
```

Comment:
let's make this code more concise, introducing the new variables is not needed if its used only one time. also please be attentive about redundant empty lines.